docs/content/Unit Testing.fsx

```diff
@@ -0,0 +1,74 @@
+﻿(**
+---
+category: end-users
+categoryindex: 1
+index: 4
+---
+
+# Unit testing an analyzer
+
+To help analyzer authors testing their analyzers there's dedicated [testing package](https://www.nuget.org/packages/FSharp.Analyzers.SDK.Testing).
+It contains easy to use utility functions to create a context for the analyzer and assertion helpers.  
+
+`FSharp.Analyzers.SDK.Testing.mkOptionsFromProject` creates the `FSharpProjectOptions` for the given framework (e.g. `net7.0`) and the given list of packages to reference.  
```

Comment:
Use more links for functions and types:
- http://localhost:8901/reference/fsharp-analyzers-sdk-testing.html (make relative URL)
- https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-codeanalysis-fsharpprojectoptions.html